codingStandards/CodingStandard-Java.md

```diff
@@ -550,9 +550,39 @@ This ensures that variables are valid at any time. Sometimes it is impossible to
 
 The concept of Java information hiding and encapsulation is violated by public variables. Use private variables and access functions instead. One exception to this rule is when the class is essentially a data structure, with no behavior (*equivalent to a C++ struct*). In this case it is appropriate to make the class' instance variables public.
 
+**9. `this.` keyword should not be used unless necessary.**
```

Comment:
I think when referring to `this` keyword, we don't have to include the period. It's not used in official sites (e.g. https://docs.oracle.com/javase/tutorial/java/javaOO/thiskey.html)